### ISSUE TITLE
Rules: express security rules

### ISSUE THREAD

**Issue by vbonvin** [vbonvin/IPT-public#8]
Ideas: 

- The IOC in general cannot be held responsible for any damage caused when working on the experiments.
- The participants can ask the LOC at the latest three weeks in advance for supplementary safety material (wooden box, safety walls, etc...). Providing or not this material is at the discretion of the LOC.
- The chairman can forbid the participants to perform an experiment live if he/she thinks the safety of the public/jury is compromised. 
- During the competition, the participants performing the experiments are held fully responsible in case of damage.

**Comment by LVoronina** [comment]
This is all about showing the experiments during the PFs? Should we add something about responsibility while they are preparing for the IPT? 

Say, if the students are preparing independently from any university/student organisation and participate for the first time, they are responsible for following the safety measures. If they are preparing as a part of a corse in the university, then they have to comply to the university regulations there or to the suggestions of the IOC representative.  Finally, if working on one of the problems violates the safety regulations of the university, the IOC representative has a right to forbid the students in his/her country working on a certain problem.  The IOC is responsible for making sure that in the list there is no more than 3 problems that might be considered as dangerous in certain countries OR which require specific equipment or weather conditions, which are available only in limited places, e.g. railways or large amounts of snow.

**Comment by vbonvin** [comment]
Yep, these are good points Lucy. 

>The IOC is responsible for making sure that in the list there is no more than 3 problems that might be considered as dangerous in certain countries OR which require specific equipment or weather conditions, which are available only in limited places, e.g. railways or large amounts of snow.

Let's say that at the stage of the preselection, each IOC representative is responsible for its own country only, and putting everything together we hope that.

**Comment by Vovcick** [comment]
Also I am not sure about the third point. If the team doesn't know about the experiment possibility in advance then it is hard to decide if they should take this problem and how should they prepare it.
I think the only solution here is to write to the participants that they ask the IOC president or secretary in advance about the possibility of a certain experiment to be held and they address this question to LOC and send the answer back to the participants. If the experiment is not allowed then the participants are to bring it on video. But if it is allowed by LOC then chairmen should be ready to accept it.

**Comment by LVoronina** [comment]
I agree with Vladimir on the last point. We should put it in the rules as well then.

**Comment by Indomerun** [comment]
@LVoronina, I think most of what you wrote can more simply be stated by saying that the participants (as well as individual IOC members) must always abide by the laws and regulations of their country and university. The best thing is probably to just state a complete waiver of responsibility for the IPT as an organisation. (As I believe @vbonvin intended)
I deem going into any further detail to be impossible, as different countries and different universities will be very different in this regard. This means that in some places the students themselves might be held responsible, in other places it might be their university or someone at the university responsible for lab safety. 

@Vovcick, the issue I see with this proposal is that it may turn into a very large amount of emails that needs to be addressed regarding simple experiments that no one would deem unsafe. We should be able to expect people to use common sense when it comes to what is safe demonstrate live or not. 
I would rather have it the way @vbonvin suggested; a statement that "the chairman can (and should?) forbid the participants to perform an experiment live ...". This way we force the participants to take such safety aspects into consideration when planning their presentations. 
But perhaps we can have it the way you suggested for experiments they feel unsure about. Thus, if they bring an experiment a chairman feels uncomfortable with but they have not checked with the IOC beforehand if it's OK to do a live demonstration, it's their problem.

**Comment by Vovcick** [comment]
I agree with @Indomerun on his comments as we should think on how to minimize the LOC work before the tournament. Let us say that Vivien's version is ok and all the participants will read it and ask in advance for permission to make some dangerous experiments if they will have such ones.

By the way we may continue this line and change the first point to "The IOC in general cannot be held responsible for anything"  :-D

**Comment by LVoronina** [comment]
:) I don’t know if the current LOC president would go for it ;)
